modules/ROOT/pages/tokenization.adoc

```diff
@@ -24,6 +24,23 @@ A connection is established between the API gateway and the tokenization service
 
 The API gateway replaces the credit card number with the tokenized data and the request is sent to the app. 
 
+
+== Prerequisites
+
+To configure and use the tokenization service, you must:
+
+* Have the Anypoint Security - Edge entitlement for your Anypoint Platform account. +
```

Comment:
Should say -- meet Runtime Fabric Pre-Reqs

Reply:
Revise to: Meet all the prerequisites and requirements for running Anypoint Runtime Fabric.